src/etc/inc/nat64.inc

```diff
@@ -0,0 +1,164 @@
+<?php
+/*
+ * nat64.inc
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2004-2013 BSD Perimeter
+ * Copyright (c) 2013-2016 Electric Sheep Fencing
+ * Copyright (c) 2014-2020 Rubicon Communications, LLC (Netgate)
+ * Copyright (c) 2020 Nick Whaley
```

Comment:
Did you copy this file from other and made few changes?  If your answer is yes, then nevermind.  If you wrote all this code from scratch you should remove BSD Perimeter and Electric Sheep Fencing copyright notices and change Rubicon line from 2014-2020 to only 2020